tensorflow/python/kernel_tests/conv_ops_test.py

```diff
@@ -319,6 +319,20 @@ def testConv2D2x2FilterStride1x2(self):
                        strides=[1, 2], padding="VALID",
                        expected=expected_output)
 
+  def testConv2D1x1FilterStride2(self):
+    expected_output = [1.0, 3.0, 7.0, 9.0]
+    self._VerifyValues(tensor_in_sizes=[1, 3, 3, 1],
+                       filter_in_sizes=[1, 1, 1, 1],
+                       strides=[2, 2], padding="VALID",
+                       expected=expected_output)
+
+  def testConv2D1x1FilterStride2Same(self):
```

Comment:
Grr, yes.  Ok, what about:

tensor_in_sizes=[1, 7, 7, 1]
filter_in_sizes=[2, 2, 1, 1]
strides = [3, 3]

If that doesn't work, forget it, and just add a TODO to add a more meaningful "SAME" test for stride > ksize.